### PR TITLE
layout components

### PR DESCRIPTION
**Pull request by peterpme**

### CODE REVIEW

File: packages/react-native-jigsaw/src/components/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+import {
+  GROUPS,
+  FORM_TYPES,
+  PROP_TYPES,
+  COMPONENT_TYPES,
+} from "../core/component-types";
+
+export function Center({
+  width,
+  height,
```
Comment (dhruska) [inline packages/react-native-jigsaw/src/components/Layout.tsx:15]: Why pass `width` and `height` as props but also a style object? Should they just live in `style`?

Comment (peterpme) [reply packages/react-native-jigsaw/src/components/Layout.tsx:15]: trying to make the api as small as possible, wdyt?

Comment (dhruska) [reply packages/react-native-jigsaw/src/components/Layout.tsx:15]: works 4 me

**Review by dhruska — APPROVED** [review]
Excited to see this!!

**Review by davesnx — APPROVED** [review]

**Pull request merged**